plugin/pkg/scheduler/algorithmprovider/defaults/compatibility_test.go

```diff
@@ -82,17 +82,123 @@ func TestCompatibility_v1_Scheduler(t *testing.T) {
 		  "kind": "Policy",
 		  "apiVersion": "v1",
 		  "predicates": [
-		    {"name": "PodFitsHostPorts"}
+		    {"name": "MatchNodeSelector"},
+		    {"name": "PodFitsHostPorts"},
+		    {"name": "PodFitsResources"},
+		    {"name": "NoDiskConflict"},
+		    {"name": "HostName"}
```

Comment:
are these the delta between 1.0 and 1.1, the full list supported as of 1.1, or just the defaults in 1.1?